pootle/apps/contact/forms.py

```diff
@@ -54,7 +54,8 @@ def from_email(self):
             self.cleaned_data['email']
         )
 
-    def recipient_list(self):
+    @staticmethod
```

Comment:
This is extending a class that had this as a regular method. So undo this change.